src/devices/Ssd1306/Command/FrameFrequencyType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ssd1306.Command
+{
+    public enum FrameFrequencyType
```

Comment:
NIT: Should we move these enums into the Commands *cs files that use them instead? I think it makes more sense if we only have one *cs file per Command.

Reply:
`FrameFrequencyType` is used in 2 commands.  Do you mean to duplicate and store `FrameFrequencyType` within each Command class that uses?

`PageAddress` is similar where it is used in 4 commands.

There are a couple of enums within Command files already, but only when they were used only for that command.  For example, `VerticalHorizontalScrollType` in `ContinuousVerticalAndHorizontalScrollSetup` class.

Reply:
I think it should be fine if we define those enums defined in one of the commands that uses them, without duplication. We do the same today with some interop classes, where we define some native structs only once in one of the PInvokes that use them, and the other files simply use that as well.

Reply:
Gotcha.  Will update.